src/Components/AddProduct/AddProduct.js

```diff
@@ -23,23 +29,18 @@ const AddProduct = (props) => {
     }
   }, [product]);
 
-  const changeHandler = () => {
+  const clearFields = () => {
     setProductName("");
     setCategory("");
     setPrice("");
     setQty("");
     setDiscount("");
     setGst("");
   };
-
-  const options = [
-    { value: "electronic", label: "Electronic" },
-    { value: "grossery", label: "Grossery" },
-  ];
-
+  
   let button = "Add";
 
-  if (props.product) {
+  if (product) {
```

Comment:
variable name should be buttonName
and you need to use the ternary operator here